docs/modules/clients/pages/java.adoc

```diff
@@ -521,7 +564,7 @@ XML::
             <ports>34600</ports>
             <ports>34700-34710</ports>
         </outbound-ports>
-        <smart-routing>true</smart-routing>
```

Comment:
Comment for visibility / reminder for me to update this later (entire page, but I'll only leave this 1 comment to not pollute the PR)